core/src/avm2/globals/flash/system/WorkerDomain.as

```diff
@@ -1,4 +1,5 @@
 package flash.system {
+    [API("680")]
```

Comment:
I think this should be `682` for Flash Player 11.4: https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/system/WorkerDomain.html